pkg/controllers/core/deployer_controller.go

```diff
@@ -331,6 +345,69 @@ func (r *DeployerReconciler) constructPodSpecForDeployer(deployer *corev1alpha1.
 	return podSpec
 }
 
+func (r *DeployerReconciler) reconcileIngress(ctx context.Context, log logr.Logger, deployer *corev1alpha1.Deployer, serviceName string) (*v1beta1.Ingress, error) {
```

Comment:
group all of the ingress related methods together, like the deployment related methods are grouped and the service related method are grouped.